src/App.tsx

```diff
@@ -10,22 +10,34 @@ import { MyTimer } from './components/Timer';
 import { usePrevious } from './hooks/usePrevious';
 import MyButton from './components/MyButton';
 import { useMyCallback } from './hooks/useMyCallback';
+import UserData from './mock/userdata';
+import { User } from './models/user';
+import MyList from './components/MyList/MyList';
 
 function App() {
   const [showTime, setShowTime] = useState(true);
   const previousShowTime = usePrevious(showTime);
   const btnContent = useMemo(() => <>show time</>, []);
+  const [userData, setUserData] = useState<User[]>([]);
 
   const changeShowTime = useMyCallback(() => {
     setShowTime((prevValue) => !prevValue);
   }, []);
 
+  useEffect(() => {
+    // Fake fetching
```

Comment:
Also is always a great idea to provide a loading state

Reply:
The Apollo way of fetching is a nice API:
```
const { loading, data, error } = useMyFetching();
loading = true or false
data = undefined or fetched data
error = undefined or new Error

Reply:
done

Reply:
This loading handle should be a custom hook nor a prop of your list